packages/wrangler/src/metrics/metrics-dispatcher.ts

```diff
@@ -19,62 +56,105 @@ export async function getMetricsDispatcher(options: MetricsConfigOptions) {
 		 *  - additional properties are camelCased
 		 */
 		async sendEvent(name: string, properties: Properties = {}): Promise<void> {
-			await dispatch({ type: "event", name, properties });
+			await dispatch({
+				name,
+				properties: {
+					category: "Workers",
+					wranglerVersion,
+					os: getOS(),
+					...properties,
+				},
+			});
 		},
 
 		/**
-		 * Dispatch a user profile information to the analytics target.
+		 * Dispatches `wrangler command started / completed / errored` events
```

Comment:
I'm a bit confused by this naming. Why do we have a `sendNewEvent()` and also a `sendEvent()`?

Reply:
renamed to sendAdhocEvent and sendCommandEvent c9ecdf82702dfe76c7867597e41b9ef39855261a

with the hope that once defineCommand is done we can combine them. tbh we could combine them now but probably not worth the effort at this stage